lib/smart_answer/calculators/coronavirus_find_support_calculator.rb

```diff
@@ -14,52 +14,72 @@ class CoronavirusFindSupportCalculator
                   :have_you_been_made_unemployed,
                   :mental_health_worries
 
-    def has_results?
-      return false if need_help_with.blank?
-
-      need_help_with != "none"
-    end
+    GROUPS = {
+      feeling_unsafe: [:feel_safe],
+      paying_bills: [:afford_rent_mortgage_bills],
+      getting_food: %i[afford_food get_food],
+      being_unemployed: %i[have_you_been_made_unemployed self_employed],
+      going_in_to_work: %i[worried_about_work are_you_off_work_ill],
+      somewhere_to_live: %i[have_somewhere_to_live have_you_been_evicted],
+      mental_health: [:mental_health_worries],
+    }.freeze
```

Comment:
This constant is good - it clearly defines the relationship between calculation actions and how they are grouped.